src/libsyntax/parse/parser.rs

```diff
@@ -651,7 +650,7 @@ impl<'a> Parser<'a> {
                 Err(err)
             }
         } else {
-            self.expect_one_of(unsafe { slice::from_raw_parts(t, 1) }, &[])
+            self.expect_one_of(&[t.clone()], &[])
```

Comment:
Please use `slice::from_ref` here.

Reply:
Changing this currently breaks compilation as it is still an unstable feature on my rustc -- should I go through with the change anyway, despite not being able to build it locally?

Reply:
You need to add a feature-gate to `lib.rs` I guess - actually, wait, cc @Manishearth @dtolnay - what does `syntex` need from `libsyntax`? Is this one of those things where it must work on stable? If so, can we at least leave a comment above this `unsafe` code saying it should use `slice::from_ref` but `syntex` must work on stable Rust?

Reply:
To my knowledge, syntex no longer has any bearing on what we can write in libsyntax. Use `slice::from_ref`.

Reply:
Interestingly, this isn't directly relevant to the issue being fixed, but I thought it'd be worth addressing an unsafe block.